frontend/src/views/Collection/index.tsx

```diff
@@ -156,6 +157,8 @@ const Collection: FC = () => {
           </StyledCallout>
         )}
 
+        {!isRevision && <Banner collection_id={collection.id} />}
```

Comment:
super nit: JavaScript uses `camelCase` instead of `snake_case` 😄 
```suggestion
        {!isRevision && <Banner collectionId={collection.id} />}
```